_layouts/redoc.html

```diff
@@ -1,7 +1,35 @@
 {% include layout/header.html %}
 
 <div class="content-container">
-    {{ content }}
+    <div id="redoc-container"></div>
 </div>
 
+<script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.41/bundles/redoc.standalone.js"> </script>
```

Comment:
@jcalcaben, will need to confirm, but I don't think we can use v2 of redoc without losing sample requests/responses.

Reply:
Gotcha, I was assuming the new version supports that, but now see that's not entirely true.